File: Data Structures & Algorithms/remove-element/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} val
     * @return {number}
     */
    removeElement(nums, val) {
        let auxIndex = 0;
        for(let i = 0; i < nums.length; i++){
            if(nums[i] !== val){
                nums[auxIndex++] = nums[i];
            }
        }
        return auxIndex;
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int uniqueValues = 1;
        if(nums.length == 0){
            return 0;
        }
        for(int i = 1; i < nums.length; i++){
            if(nums[i] != nums[i-1]){
                nums[uniqueValues++] = nums[i]; 
            }
        }
        return uniqueValues;
    }
}